### PR TITLE
OIL-89: optimize js

### PR DESCRIPTION
**Pull request by subsend**

### CODE REVIEW

File: src/sandbox/components/cmpCallGetConsentData.html [hunk 1]
```diff
@@ -38,9 +38,9 @@ <h1 class="display-6">CMP Api Call <code>getConsentData()</code></h1>
 <script type="text/javascript">
   function invokeGetConsentData(consentStringVersion) {
     __cmp('getConsentData', consentStringVersion, (result, success) => {
-      document.getElementById('getConsentData-command-success').innerText = success;
-      document.getElementById('getConsentData-command-result').innerText = JSON.stringify(result, null, '  ');
-      document.getElementById('getConsentData-command-resultTable').style.display = 'block';
+      $('#getConsentData-command-success').text(success);
+      $('#getConsentData-command-result').text(JSON.stringify(result, null, '  '));
+      $('#getConsentData-command-resultTable').css('display: block');
```
Comment (phogel) [inline src/sandbox/components/cmpCallGetConsentData.html]: I thought changing css in jquery would require two parameters? Like so `.css('display', 'block');`

Comment (subsend) [reply src/sandbox/components/cmpCallGetConsentData.html]: we do not use jQuery but nanojs https://github.com/vladocar/nanoJS

Comment (phogel) [reply src/sandbox/components/cmpCallGetConsentData.html]: 👍

Comment (ltparis2018) [reply src/sandbox/components/cmpCallGetConsentData.html]: Hm, that means the syntax is like jQuery but not exactly the same? That could be confusing, I fear...

**Pull request merged**